Notes are from "Molecular Biology made simple and fun" by D.P. Clark and L.D. Russell, 2010, 4th edition. The following is from the 2000 version. Outdated:  there is implication that there are ~50,000 genes in higher organisms, but the latest estimate: 
19K to 20K protein coding genes in humans. So, will be edited soon. 

Prokaryote - Bacterial cells, simplest living cells. Have a single chromosome, and no separate nucleus and cytoplasm as with higher
organisms (Eukaryotes). DNA is inside the nucleus in Eukaryotes. Bacterial cell is 1 micron long and 0.5 micron wide (typ). Bacteria 
have two names, both printed in italics: Genus (a group of closely related species) and species (the individual name). Eg., *E.coli*

**Some conventions to pracice**: Write gene names in italics ( e.g., *lacZ, lacY, lacA*, the three genes that make up the *lac* operon in *E.Coli*). 

Most bacteria are beneficial. Some disease causing ones: Cholera, TB, plague, anthrax, syphillis, Whooping Cough, diptheria, etc. 
Most antibiotics are synthesized by a kind of fungus known as mold. Viruses (bacteriaphages) can infect (and takeover) bacteria. Viruses are packages of genes in protein coats. They need a living host to multiply. Some human viral diseases: polio, smallpox, herpes, Ebola, and AIDS. 

3. Genetic terms: Each gene may exist in alternate forms (eg. red or white flowers) - called alleles. Protein is a polymer made from AA (amino acids); they make up most of the structures in the cell and do most of the work. Enzyme is a protein which carries out a chemical reaction. Mutation - is an alteration in the genetic info carried by a gene. wild-type: the original or nautual version of a gene or organism. Null allele: a mututed version of a gene which completely lacks any activity. Outward characteristic - is the phenotype, while the genetic make-up is the genotype. Genes consist of DNA which are arranged on chromosomes. Bacteria have only one copy of each gene ('haploid'). Higher organisms carry duplicate copies ('diploid'). If more than two copies -- triploid, tetraploid, etc. When two different alleles (Rr) are present, the one that dominates (phenotype-wise) is the dominant allele (the other one - recessive allele). When both alleles are identical - homozygous (RR or rr) ; different - heterozygous (Rr). When two animals or plants reproduce, the mother makes egg cells and the father makes sperm cells. These reproductive cells are germ-line or gametes. Zygote cell formed by the union of sperm and egg develops into a new individual. Somatic cells make up the body. Somatic cells are diploid; egg and sperm cells  are haploid. The Diploid somatic cell undergoes meiosis to give haploid germ cells. Filial generations - successive generations of descendants from a genetic cross (F1, F2, etc). Posession of two X chromosomes makes one  femaie; XY makes a male. A single copy of an allele may be partially dominant; two functional alleles may be co-dominant. Genes are linked if on a single copy of a chromosome. Breaking and rejoining of DNA strands may lead to crossing-over. The genetic result of shuffling different alleles: reconbination. The farther apart two genes are on the chromosome, the more likely is a crossover - and higher frequency of recombination. 

4. Molecular basis of heredity: Genes are made of DNA (deoxyribonucleic acid). Each gene is a linear segment of a long DNA molecule. DNA is a polymer - which is a linear arrangement of nucleotides. Each nucleotide has 3 components: a phosphate group (p) , a sugar (dR or R), and a nitrogen-containing base. In DNA, the sugar is deoxyribose (dR). 4 types of bases: Adenine, Thymine, Guanine, and Cytosine. The bases are joined by Deoxyribose (dR) and stick out sideways. P and dR form the backbone. Nucleotides are joined by linking p on the 5' end of dR1 and 3' end of dR2. The numbering: carbon atoms on a dR are numbered 1' thru 5'. DNA is normally found as a double stranded molecule. Helical arrangement - double helix. A of one strand pairs with T of the other; same with G and C. Each base pair has one double size purine base (A and G - they contain a double ring called purine ring) and a smaller puridine base (C and T). Widths are the same for all. Hydrogen bonds are formed between base pairs. GC have 3 hydrogen bonds (vs 2 of AT) and can withstand higher temperatures. Hydrogen bonds are weak; but a DNA molecule has ~Mbp which makes the bond strong. The bonds use nitrogen or oxygen on opposite sides. mRNA (messenger RNA) molecules (short sequences) are formed from the DNA strands and pass through the nuclear membrane pores to the plasma for protein formation. The genes are bound to proteins in the nucleus and together they form the visible material (chromatin). An average bacterial cell is 1 micron long; but the DNA molecule is 1 mm long (1000 times longer). It fits inside the cell coiled (supercoiling). They typically have 50 giant loops of supercoiled DNA arranged around a protein scaffold. 

5. Duplicating the DNA: replication. Each chromosome must be accurately copied for inheritance. First, both the supercoils and the double helix must be unwound to separate the two strands (by DNA gyrase and DNA helicase, respectively). Single strand binding proteins (SSB) bind to individual strand bases and prevent them from coming together again. Each parental strand is a template strand. The incoming bases connect individually to the strands. Hydrogen bonding is good 99% of the time. DNA polymerase III (Pol III) improves this accuracy (rejects mismatches) and enjoins them. Synthesis always goes from 5' to 3'. The carbon atoms of dR are numbered with prime marks (the base carbons do not have primes). The two complementary strands run in opposite directions (the other one: 3' to 5'). They are said to be antiparallel. The strands are synthesized in opposite directions. One strand, 'leading strand', is made continuously, while the other strand is only made in short segments ('lagging strand'), since DNA and RNA are always synthesized 5' to 3'. The lagging strand pieces are named Okazaki fragments (after their discoverer); they are joined by DNA polymerase I (to fill in the gaps) and DNA ligase (to bind the ends). How does a new strand start: a primase enzyme lays down a short RNA primer for this. Two double helical DNA molecules are produced, each with one old and one new strand. Cell Division: Eukaryotic chromosomes are linear (not circular as with bacteria), long, and has several replication forks. Phases: DNA synthesis (or S) phase; two gap processes (no function phases), before and after Mitosis (which takes < 1/12th the total period vs S phase >5/12th). Mitosis has 4 phases: pro,meta, ana, and telophase. 

6. messenger RNA (**mRNA**): Genetic info is carried in the nucleic acid molecules of DNA and RNA. RNA carries a working copy of the genes. **Transcription**: genetic info from DNA is converted to its RNA equivalent (mRNA). DNA sugar is deoxyribose (one less oxygen vs ribose); for RNA it is ribose. One base is different: RNA has U (uracil) in place of DNA's T (thymine). RNA is single stranded vs DNA's double strand. RNA is complementary to the 'template' strand, and hence is identical to the 'code' strand of DNA (but for U vs T). 20 to 30% of the genes are used in a given cell. Each gene, or a small group of related genes, generate a separate RNA copy if and when needed. RNA polymerase, an enzyme, binds to DNA at the start of a gene and opens the double helix. The 'start' is recognized by the -10 region and -35 region (counting backwards from the first base of the gene), respectively the promoter and upstream regions. At -10, the seq is TATAAT and at -35, it is TTGACA. These are consensus sequences, typically exact for nearly all highly expressed genes, but close enough for others. The RNA polymerase stops making RNA when it encounters a terminator (palindromic) sequence (made of two inverted repeats separated by 6-8 base, followed by a run of As). The second repeat is the complement of the inverse of the first. This inverted pair causes a stem and loop or 'hairpin' type of structure (when these repeats bind together). The string of As in DNA gives a string of Us in the corresponding RNA. When RNA polymerase reaches the hairpin, it pauses. The mRNA created so far may fall off due to poor bonding between the As and Us; the RNA polymerase also falls off and can start another transcription. 

    Which genes turn on: Some genes (housekeeping genes) are turned on always. For other genes (or group of genes), a gene activator protein specific to the gene and the tissue binds first to the -35 region and helps guide RNA polymerase to that site. This activator is typically activated by an outside molecule. Eg. E. coli has an **activator** protein MalT that detects and binds to maltose, a type of sugar. This changes its shape and it can now bind to the DNA sequence for a gene that can digest maltose. That is, this gene is only expressed when this particular sugar is available.  A **repressor** protein turns a gene off. Eg. Lactose repressor LacI, when there is no lactose to consume,  binds to the operator sequence (-5 to +21) near/over the gene that can digest Lactose. Thus the gene is turned off. When lactose is available, LacI changes shape and falls off the gene, allowing the RNA polymerase to turn it on. Most of such regulator proteins change shape when they bind to their specific signal molecule (because they change shape, are called as allosteric proteins).

    They may have two or four subunits (with the other set binding the inverted repeat site - because of the twisting in the double helix, the template and coding strands may be on the same side in space. Fig .6.16). Crp (Cyclic AMP receptor protein) is an example of a **global control protein** used in controlling and coordinating the work of  a large group of genes. It selects the nutrient for growth in E.Coli - when Cyclic AMP signal is received/generated by the cell (when it has run out of glucose). This coupled with the specific regulator (such as LacI) would then be able to select a specific (here, lactose) molecule for nutrition. Cyclic is a circular version of AMP. It is called a 'regulatory nucleotide.' In E. coli,  a global promoter can turn on a cluster of genes ('operon'), with the individual promoter/repressor involved in fine tuning. No such operon in Eukaryotes. Antisense RNA, if made, can also switch off a gene. It is made, not with the non-template DNA strand, but an anti-gene sequence in the template strand. Bacteria demonstrate quorum sensing - when billions are crowded together, their secretion of a signal molecule into the medium will help them coordinate. Example: Enzyme luciferase in *Vibrio fischeri*, a cousin of *E.coli* can help them glow together in the sea - while on rotting organic matter; a fish will swallow them, thus giving them their gut as a place to live in. 

7. Proteins: types - structural, regulatory and transport proteins and enzymes. Proeteins are made from a linear chain of monomers (Amino acids) and folded into complex 3-D shapes. A (polymer) chain of AA is a 'polypeptide chain.' Proteins can have one or more such polypeptide chains. Structural proteins are used for creating subcelllular structures.  An enzyme carries out a chemical reaction - it will need a cleft in a 3-D wound protein which has brought (near and) together different AA sequences to build a strong bond with the enzyme; a regulatory protein controls the expression of a gene or the activity of another protein; and a transport protein (found mostly in biological membranes) carries other molecules across membranes or around the body (eg. Hemoglobin). Enzyme example- beta-galactosidase splits lactose into two simple sugars (glucose and galactose). Cofactors or prosthetic groups (single metal atoms or more complex molecules) aid proteins. A protein without its prosthetic group is an apoprotein. Eg. Hemoglobin comprises of the apoprotein globin and has an x-shape cofactor that holds an iron atom ('heme'). Oxygen binds to the iron atom and hemoglobin carries it around the body. 'heme' is shared by hemoglobin and myoglobin. 

    Protein construction: 20 different AAs. an AA has a central carbon atom, the alpha carbon, surronded by four groups: NH2 (amino), central hydrocarbon (inclusive of the alpha), COOH (carboxyl), and R or variable group. AAs join by peptide bond (OH of COOH of one and H of NH2 of the next combine;  a water molecule is formed which is lost. Many such bonds are formed, eventually with an N-terminus (NH2) on the LHS (left hand side) and a C-terminus (COOH) on the RHS (right hand side). The polypeptide chain forms a 3-D structure thus: (a) The order of the manomers is the 'primary structure.' (b) A secondary structure is formed via hydrogen bonding (In DNA hydrogen bonding leads to the helical double strand structure)- two types (alpha helix -hydrogen bonds are formed vertically; the beta-sheet bonds peptide groups, horizontally. (c) a tertiary structure that is a supercoil (eg. in circular DNA, supercoiling causes it to become a figure of 8). Many 3-D conformations occur (as there are 20 AAs). Most proteins are roughly spherical. The 3-D folding results from R groups that are hydrophilic (and hence on the outside of the protein, so the protein can dissolve in the water surrounding) and hydrophobic (and hence away from the water). Hydrophobic molecules are greasy and insoluble, they are called 'oil drop model.' (d) A quaternary structure if more than one polypeptide chain. There are 20 AAs, represented by 3-letter code and 1-letter code (all English alphabets, but for BJOUXZ). How are proteins made ('central dogma' of molecular biology, from Sir Crick): **Transcription** - DNA to mRNA; and **Translation**: mRNA to Amino acids that make up a polypeptide chain (translates from DNA to AA language). Usually one gene in the DNA yields one protein. During translation, the bases of mRNA are read in groups of 3 (codons). Each codon represents an AA. 64 possible combinations of 3-bases (4 to the power of 3) - so some AAs are encoded by more than one codon. AUG (codes for M or Met also) is the start codon, with UAA, UAG, and UGA as stop codons. From Wiki: In eukaryotes, there are only 21 proteinogenic amino acids, the 20 of the standard genetic code, plus selenocysteine (from UGA, normally a stop codon). Humans can synthesize 12 of these internally. The other nine must be consumed, and are called essential amino acids. The essential amino acids are histidine, isoleucine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, and valine (i.e. H, I, L, K, M, F, T, W, V). Meat contains all these amino acids. Four essential AAs missing in most plant foods are: Lysine, Tryptopha, Methionine, and Phenylalanine. Certain vegetarian foods contain all the amino acids: Quinoa, Soy, Chia seeds, and Spinach. Vegetarians should incorporate these in their diets. 
    
    Ribosomes (made up of ribosmal RNA or **rRNA**) do the decoding. The book discusses bacterial protein synthesis here. The ribosome binds to the mRNA, moves along it, adding a new AA to the growing polypeptide chain each time it reads a codon. **tRNA** (transfer RNA) have at one end an anticodon (3 bases that are complement to the 3 codon bases). Codon and anticodon bond via hydrogen. At the other end, tRNA carries the AA that are strung together into a long polypeptide chain (note from Wiki: Essential amino acids cannot be made by the body. As a result, they must come from food. The 9 essential amino acids are: histidine, isoleucine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, and valine. Nonessential amino acids are produced in the body. The pathways for the synthesis of nonessential amino acids are quite simple. More in the previous paragraph). There are 20 to 64 different tRNAs (some tRNAs can read more than one codon that code for the same AA - wobble rules allow the first base to be different). Eg. tRNA for histidine, with GUG as the anticodon, can recognize both CAC and CAU codons. Exceptions: Book discusses I (inosine) base in an anticodon.   The book uses a sequence to discuss 'Open Reading Frame' (**ORF**). Three possible start codons (AUG) exist in that sequence. AUG codes for Met AA. Which one? Near the 5' end, there is a ribosome binding site - the S-D sequence. The complementary sequence is found near the 3' end. The start codon is the next AUG codon after this ribosome binding site (AGGAGGU or UCCUCCA). Mostly that simple. Peptide chain grows - Page 77 and 78. As the chain grows, tRNA are released and reused later. Elongation Factors, EF-T and EF-S, a pair of proteins each, are involved in sideways shuffling of the tRNAs on the ribosome. Energy is needed. For termination, release factors read the stop signal (UGA, UAG, or UAA) and chop the completed polypeptide chain off of the final tRNA. One mRNA can code for many proteins in bacteria. In higher organisms, instead of S-D sequence, the front end of the messenger is recognized and only the first ORF is translated. Several ribosomes can build the chain at the same time, perhaps 100 bases apart. This process is called polysome (for polyribosome). tmRNA (it does both transfer and messenger RNA functions) helps complete a stalled protein chain (possibly the result of a defective base in mRNA) buildbup and complete the process. It will also tag the defective polypeptide for destruction. Author suggests that tmRNA implies: 'terminate me' RNA! 

7. NHGRI has up-to-date information [here](https://www.genome.gov/10000202/fact-sheets/). I will make notes from here for the time being. Centromere: the constricted region of the linear chromosomes; typically located near the center, but on occasion at one end. Telomeres are repetitive DNA sequences located at the end of linear chromosomes; most cells lose a bit of telomeres when they divide; WBC and cancer cells do not. Females have two X chromosomes in their cells; males XY. International HapMap project developed a haplotype (A haplotype is a set of DNA variations, or polymorphisms, that tend to be inherited together) map of the human genome. **HapMap** is used to find genes affecting health, disease and responses to drugs and environment. People share 99.9% of their DNA sequences; the remaining 0.1% has genetic variants that change disease risk and drug response. SNP Vs SNV: SNP refers to a more common variation that is seen across a subset of the population. SNV is a variation at a single nucleotide with no population metric attached to it. About 10 M SNPs are there in humans; rarer SNP allele are in >1% population. A set of associated SNP alleles in a region of a chromosome is a 'haplotype.' A chromosome has a few common haplotypes. We can use a few tag SNPs to id the whole pattern of genetic variation. HapMap provides these common patterns. About 0.5 M tag SNPs provide the same disease info as searching all of 10 M SNPs. Current HapMap is based on 269 people, representative of various international populations. For some reason, it is believed that all haplotypes with frequencies of 5% or higher have been identified. A tall claim? **Note**: "the project's policy does not prevent researchers from applying for patents on SNPs or haplotypes for which they have demonstrated a specific utility." The following biological pathways are better understood: metabolism, gene regulation (turning on and off), and signal transmission (through cellular surface receptors). A biological network has multiple pathways that interact with each other. Knowledge of these pathways may lead to more personalized strategies for diagnosing, treating and preventing disease. Unlike previously thought, cancer is not driven by single genetic mutations and understanding of biological pathways that are disrupted by these genetic (SNP) mutations. Chromosome abnormalities: numerical - monosomy - a missing chromosome; trisomy - more than 2 chromosomes; structural - deletion, duplication, translocation (to another chr), inversion, ring. 'de novo' is a chromosome abnormality new to the person, not inherited. 'Cloning': different processes to produce genetically identical copies of a biological entity. types; gene, reproductive, and therapeutic. 
8. Fact sheets from NHGRI continued: Comparative genomics - conserved sequences (over mya) helps understand the genome itself (and the biological processes) and evolution. Example: 2/3rds of human oncogenes have counterparts in the fruit fly. NHGRI programs **ENCODE** (Encyclopedia of DNA Elements) and **modENCODE** (model organism Encyclopedia of DNA Elements) help better understand how genomes function. DNA Microarray use: As an example, researchers have found >800 SNPs in BRCA1. A chip will contain thousands of short, synthetic, single-strand DNA sequences, which together add up to the normal gene in quesion and its variants. Now array used for gene expression switch state (RNA used here), clinical diagnosis and drug efficacy for some diseases. GWAS: Useful for finding genetic varitions that contribute to common, complex diseases, such as asthma, cancer, diabetes, heart disease and mental illnesses. Tools useful, from Genome project (2003) and HapMap project (2005) - computerized dbs with ref human genome sequence, a map of human genetic variation and a set of tools to analyze WGS for genetic variations. Database of Genotype and Phenotype (dbGaP) is [here](https://www.ncbi.nlm.nih.gov/gap). I found one study relevant to my work: Heart and Vascular Health Study [(HVH)](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/variable.cgi?study_id=phs001013.v3.p2&phv=259387&phd=5482&pha=&pht=5311&phvf=&phdf=&phaf=&phtf=&dssp=1&consent=&temp=1)
9. Fact sheets from NHGRI continued: Genetic (or linkage) mapping - successfully used for relatively rare signle-gene inherited disorders. Blood or tissue samples are collected from families and sequenced to find SNPs. A gene close to the DNA markers (SNPs) is likely to stay together through recombinations, and this gene gets studied. HGP has found dense maps of markers across the entire human genome. This can be viewed at the [NCBI](https://www.ncbi.nlm.nih.gov/) site as well as the [USCS](http://genome.ucsc.edu/) site. The NCBI site has multiple pages that has information categorized (but there is overlap in information) on DNA/RNA, Domains & Structures (BioSystems, Conserved Domain for protein production, and 3-D protein structures), Genes & Expressions, Genetics & Medicine, Genomes & Maps, Homology, Literature, Proteins, Sequence Analysis, Taxonomy, Training & Tutorials, and Variation. Check out the Downloads tab for appropriate tool and db downloads. There is also a separate tab for Databases. Glossary is listed [here](https://www.ncbi.nlm.nih.gov/books/NBK21106/). I will document a relevant few here.
10. Glossary: 
    * Alu - They are SINES (short interspersed elements) present in multiple copies in human and other primate genomes; about 300 bp in length, found in introns, 3' UTR, and intergenic genomic regions; mobile elements and has most abundant copy number (0.5 M). It is used as a universal PCR primer as it binds in both directions. 
    * domain - a discrete portion of a protein assumed to fold independently and has its own function
    * enhancer - enhance the rate of transcription; typically they are far away (1000s of bp away) from the sequence. 
    * GeneID - unique ID in Entrez Gene; an integer which is species specific. Reported in RefSeq as a 'db_xref'. 
    * homologous - similarity attributable to descent from a common ancestor
    * Map Viewer - available in [Entrez Genomes](https://www.ncbi.nlm.nih.gov/genome). Can view the complete genome, display chromosome maps,and zoom in. 
    * microsatellite - repititive stretches of short DNA sequences used as genetic markers to track inheritance in families. Also known as STRs (short tandem repets).
    * orthology - describes different species that derive from a common ancestor
    * paralog - a set of homologous genes that have diverged. 
    * PDB - [Protein Data Bank](https://www.rcsb.org/)
    * promoter - A transcription factor attaches to this sequence to initiate transcription. Consists of 3 regions: Initiator box where transcription starts; TATA box 25 bases upstream from there; and 10 bp long upstream element - specific to a protein. 
    * Pseudogene - defective duplicate copy of a genuine gene; cannot be expressed. 
    * RNA Polymerase - binds to DNA during transcription and separates or unwinds the DNA strands. Three types and genes that they transcribe: I - genes for large ribosomal RNAs, II - genes that code for proteins, and III - genes for tRNA and other tiny RNAs. 
    * QTL - Quantitative Trait Locus - a hypothesis that a certain small chromosomal region contributes significantly to a complex trait expression (based on density of linkage map)
    * RefSeq -  non-redundant reference sequences. 
    * RepeatMaker - [program](http://ftp.genome.washington.edu/) that screens DNA sequence for interspersed repeats and low-complexity DNA sequences. 
    * splicing - cutting out the intronic regions. Telomere cap and poly-A tail are added prior to this. 
    * transcription factors - Proteins that regulate RNA Polymerase II, by binding to certain specific DNA sequences,viz., promoter and enhancer
    * UniGene cluster - sequences for known or putative genes. Complete dataset is [here](ftp://ftp.ncbi.nih.gov/repository/UniGene/)
    * WGS sequence - Whole Genome Shotgun sequence - uses large arrays of sequences to sequence rapidly the sequence.
11. Chapter 11 on Genetic Org of Eukaryotes: In readme2.md. (Continued here with the 4th edition of Clark and Russel's book. Yet to validate all earlier material. Figure 11.28 shows a mature eukaryotic mRNA with a cap, exons, and a tail. In the ribosome, the initiator tRNA (finds the start codon) and release factor (ids the stop codon) are involved. More details to be added later).
12. Chapter 12, Mutations: *silent* mutations occur in the non-coding DNA between genes, in introns, and even in the exonic regions (since most of the 20 possible AAs have more than one codon). *Missense* mutation - a codon is altered, so one AA in a protein is replaced with a different AA. Active site - special site of pocket on a protein where the chemical reaction occurs. If the missense mutation occurs here, then the protein action is impacted. Since this is only a small portion of the total protein sequence, most changes are mild and non-lethal. Called 'conservative substitutions. Figures 12.7 and 12.8 show the impact of conservative and radical change. Fig 12.7, GCA --> GGA (Ala --> Gly). Non-lethal. Fig 12.8: GCA --> GAA  (Ala to Glu). Radical replacement --> Gla is gluatamic acid, which is acidic and carries a strong negative charge. The mutated protein  does not fold properly. So, the protein action is compromised. Temperature sensitive mutation: eg. - black tips in some animals' paws and tails. They are warm-blooded animals. At normal temps, melanin is not made over most of the body; the paws are cooler, melanin is made, and hence the black coloration. Pretty mutation. *Nonsense* mutation: UCG --> UAG (seine --> sTOP codon). The rest of the protein does not get made. Hence nonsense mutation. Deletion: when taken 3 bases at a time, the reading frame is changed when a base is deleted. It is called frameshift mutation. Deletions or insertions of two bases also cause reading frame changes. 3 base deletion/addition: if in a less vital region, no problem. Inversions and translocations: inversion - when a segment of DNA is removed, flipped and reinserted, facing in the opposite direction. Translocation - when a section of DNA is removed from its original position and moved to another location, perhaps on another chromosome - not a problem if the whole gene is moved. Induced mutations are caused by mutagens - toxic chemicals and radiation. Spontaneous mutations are due to errors in DNA replication. Transposon - mobile segments of DNA that moves from place to place on a cell's chromosome. These are distinct from short insertions described above. Hot spots for mutations: Are due to the presence of methlycytosine bases in the DNA. Mostly they pair correctly with G, but may spontaneously disintegrate to U, which pairs with A. Ames test is used by industry and government agencies to screen chemicals for possible mutagenic effects. A specific bacterium, carrying mutation for histidine synthesis are used. If mutations occur when large number of these mutant bacterial are placed in a growth medium lacking histidine, then mutagenic effect is high. 
13. Inherited Human Diseases: Ch. 13 of the Clark and Russel book - (In higher animals) - somatic cells have no long term future. When the body dies, they die as well. Only the germ line cells pass their genes to the next gen. In most cases, detrimental mutations are recessive to the wild-type. That is, single good copy of the gene is enough for normal function and will mask the defective copy. Children are homozygous for the recessive allele if both parents are close relatives (and both carry that defective copy). Eg. high incidence of hemophilia among the royal families of Europe. Hence the taboo against incest inn most societies. Table 13.1  lists best known single gene genetic defects- are recessive and require both copies to be defective before symptoms appear: CF, MD, Hemophilia, Sickle Cell Anemia. Multiple Gene defects: Down Syndrome, 1 in 800, due to an extra copy of chromosome 21. Chance increases with maternal age. 

